architecture/additional_concepts/authorization.adoc

```diff
@@ -880,6 +880,42 @@ in their SCC set.  This allows cluster administrators to run pods as any
 user by without specifying a `RunAsUser` on the pod's `SecurityContext`.  The
 administrator may still specify a `RunAsUser` if they wish.
 
+[[rbac-to-scc]]
+==== Role-Based Access to SCCs ====
+Starting with version 3.11, you can specify Security Context Constrains as a
+resource that is handled by your roles or cluster roles. This allows you to
+scope the access to your SCCs for users/groups locally to a certain project
+compared to previously assigning users/groups to an SCC, which had a
```

Comment:
replace with.
Assigning directly user/groups to an SCC retains cluster-wide scope